changelogs/fragments/451-libssh-remove-tech-preview.yaml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
```

```suggestion
bugfixes:
```
Sorry for waffling on this when this is what I told you to do earlier. I don't think there's really a good section to put this under but it does need to get called out. Docs bugfixes are still bugfixes though.